servers/intro_course/infrastructureSetup/gitlab.go

```
@@ -0,0 +1,354 @@
+	}
+
+	_, _, err = git.Boards.CreateIssueBoardList(project.ID, issueBoard.ID, &gitlab.CreateIssueBoardListOptions{
+		LabelID: gitlab.Ptr(IN_REVIEW_LABEL_ID),
+	})
+	if err != nil {
+		log.Error("failed to create issue board list: ", err)
+		return errors.New("failed create issue board list")
+	}
+
+	// Add issue board labels
+	_, _, err = git.Tags.CreateTag(project.ID, &gitlab.CreateTagOptions{
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)